assets/js/modules/tagmanager/setup.js

```diff
@@ -486,15 +544,20 @@ class TagmanagerSetup extends Component {
 						onEnhancedChange={ this.handleAccountChange }
 						outlined
 					>
-						{ accounts.map( ( account ) => {
-							return (
-								<Option
-									key={ account.accountId /* Capitalization rule exception: `accountId` is a property of an API returned value. */ }
-									value={ account.accountId /* Capitalization rule exception: `accountId` is a property of an API returned value. */ }>
-									{ account.name }
-								</Option>
-							);
-						} ) }
+						{ []
+							.concat( ACCOUNT_CHOOSE === selectedAccount ? { accountId: ACCOUNT_CHOOSE, name: __( 'Select one...', 'google-site-kit' ) } : [] )
```

Comment:
I think this should always be included, both for accounts and containers. Removing it once you've selected an account is unexpected, especially for accessibility.